BrickBrackBruck/WalterDerHalter.cs

```diff
@@ -1,23 +1,121 @@
 using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+
 
 namespace BrickBrackBruck
 {
+
     class WalterDerHalter
     {
-        // List<
+        List<Ding> dinger = new List<Ding>();        
+
+        DateTime tag0 = new DateTime(1990,01,01);
```

Comment:
das ist n bisserl brutal - einfach auf 1990 setzen
entweder die Arrays dynamisch erweitern oder das Einlesen der Dateien in 2 Phasen:
Phase 1 - Meta-Daten der Dateien einsammeln (bspw. Startzeitpunkt)
Phase 2- alle Dateien einlesen, parsen und verarbeiten